cdap-app-templates/cdap-etl/cdap-etl-core/src/test/java/co/cask/cdap/etl/planner/ConnectorDagTest.java

```diff
@@ -40,12 +40,10 @@ public void testMergeConnectors() {
       .addConnection("n2", "n3")
       .build();
     cdag.insertConnectors();
-    // n3 should have a connector inserted in front of it
+    // no connector should be inserted since we support multiple inputs in a dag
```

Comment:
perhaps a better name for this test case now would be testMultipleSourcesIsNoOp() or something along those lines